src/test/java/teammates/test/cases/logic/EmailGeneratorTest.java

```diff
@@ -205,22 +205,56 @@ public void testGenerateFeedbackSessionEmails() throws IOException {
         
         ______TS("feedback session submission email");
 
-        Calendar time = Calendar.getInstance();
-        time.set(Calendar.DATE, 4);
-        time.set(Calendar.MONTH, 8);
-        time.set(Calendar.HOUR_OF_DAY, 5);
-        time.set(Calendar.MINUTE, 30);
+        Calendar timeForStudent = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+        timeForStudent.set(Calendar.DATE, 4);
+        timeForStudent.set(Calendar.MONTH, 8);
+        timeForStudent.set(Calendar.HOUR_OF_DAY, 5);
+        timeForStudent.set(Calendar.MINUTE, 30);
         EmailWrapper email = new EmailGenerator().generateFeedbackSubmissionConfirmationEmailForStudent(
-                session, student1, TimeHelper.formatTime12H(time.getTime()));
+                session, student1, timeForStudent);
         subject = String.format(EmailType.FEEDBACK_SUBMISSION_CONFIRMATION.getSubject(), course.getName(),
                                 session.getFeedbackSessionName());
         verifyEmail(email, student1.email, subject, "/sessionSubmissionConfirmationEmailForStudent.html");
 
+        Calendar timeForInstructor = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
```

Comment:
Just use one `Calendar` instance, like how it is done before. It makes manual checking easier: keep the time in the `Calendar` instance in memory and work out the timestamp in the email using the session's timezone.